person.rb

```diff
@@ -24,4 +24,8 @@ def can_use_services?
   def of_age?
     @age < 18
   end
+
+  def add_rental(book, date)
+    Rental.new(date, book, self)
+  end
```

Comment:
- Good job done so far, but you have not yet established the `has-many` relationship between the `Person` class and the `Rental` class. Please add a `@rentals=[]` instance variable to fix this 🙏. Dont forget to implement both setter and getter methods for the rentals variable 👍 👍 
- Also, please pass in the argument for the rental class in the correct order, your Rental class takes in the date, person, and book, in this case, self, in that order. Please fix this to avoid any error